src/components/course-calendar/month.cjsx

```diff
@@ -38,8 +45,8 @@ CourseMonth = React.createClass
     nodesWithHeights = calendar.querySelectorAll('.rc-Week')
 
     Array.prototype.forEach.call(nodesWithHeights, (node, nthRange) ->
-        range = _.findWhere(groupedDurations, {nthRange: nthRange})
-        node.style.height = range.dayHeight + 'rem'
+      range = _.findWhere(groupedDurations, {nthRange: nthRange})
+      node.style.height = range.dayHeight + 'rem'
```

Comment:
Can you document in the LESS the node that will get this height? and include a comment in here with a greppable string to be able to find it easily? Something like `CALENDAR_DAY_DYNAMIC_HEIGHT`? There seem to be more of these special-style cases showing up in the code and I'm worried it'll be difficult to find them in 6mos.